internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -467,21 +513,20 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) bedrockStopReasonToOpenAI
 
 func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) bedrockToolUseToOpenAICalls(
 	toolUse *awsbedrock.ToolUseBlock,
-) *openai.ChatCompletionMessageToolCallParam {
+) openaigo.ChatCompletionMessageToolCall {
```

Comment:
switch to a pointer return?

Reply:
we just dereference the ptr right after returning, what do u think is best to do?